src/Services/PageService.cs

```diff
@@ -42,10 +53,24 @@ public PageService()
             }
         }
 
-        var experimentalFeaturesVM = App.Current.GetService<ExperimentalFeaturesViewModel>();
-        foreach (var experimentId in App.NavConfig.ExperimentIds ?? Array.Empty<string>())
+        var experimentationService = Application.Current.GetService<IExperimentationService>();
+        ExperimentalFeature.ExperimentationService = experimentationService;

```

Comment:
Nit: condense to one line
```suggestion
        ExperimentalFeature.ExperimentationService = Application.Current.GetService<IExperimentationService>();
```

Reply:
It's also used below and I thought it would be weird to target the one on ExperimentalFeature.